### PR TITLE
Emaciated animals can no longer revive as zombies

### LINKED ISSUE #61999: helpless dog and twitching spider revive as woodland wight
### Describe the bug

twitching spider and helpless dog zombifies into woodland wight, possibly because they `copy-from` the motionless deer.

### Steps to reproduce

1. Make a new world
2. Spawn in twitching spider/ helpless dog
3. Kill them
4. Wait until revived
5. Zeer.

### Expected behavior

Not reviving into zombie deer, unless this is something the blob can do.

### Screenshots

_No response_

### Versions and configuration

- OS: Android
    - OS Version: Manufacturer: samsung; Model: SM-A710F; Release: 7.0; Incremental: A710FXXU2CRC2; 
- Game Version: deff4fc [32-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    No Fungal Growth [no_fungal_growth],
    Bionic Professions [package_bionic_professions],
    Misc Sprites [misc_sprite]
]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by MNG-cataclysm**
#### Summary
Bugfixes "Emaciated animals can no longer revive"

#### Purpose of change
The twitching spider, motionless deer, and helpless dog can no longer revive as zombies, as I thought they would be in too poor a condition to zombify into something. Parasitoid wasps, such as the dermatik, tend to kill the host by eating them before emerging. With the size of dermatik larva, this would leave a shell of a corpse with very little to revive.
This will also fix #61999 

#### Describe the solution
I removed the "copy_from" field from the motionless deer, and migrated what was needed, excluding the "zombifies_into" object. 

#### Describe alternatives you've considered
Having the emaciated animals revive into something else.

#### Testing
I spawned all three monsters and killed them. None of them revived.

#### Additional context
The changes made to the exodii crashes were to appease an unrelated test error.

### CODE REVIEW

File: data/json/monsters/insect_spider.json [hunk 1]
```diff
@@ -1804,19 +1804,35 @@
   {
     "id": "mon_dermatik_incubator_deer",
     "type": "MONSTER",
-    "name": { "str": "motionless deer" },
+    "name": { "str_sp": "motionless deer" },
```
Comment (MNG-cataclysm) [inline data/json/monsters/insect_spider.json:1807]:
```suggestion
    "name": { "str": "motionless deer" },
```

**Comment by NetSysFire** [comment]
I think you could keep the copy-from and instead specify `zombifies_into: "mon_null"`

**Comment by MNG-cataclysm** [comment]
> I think you could keep the copy-from and instead specify `zombifies_into: "mon_null"`

I tried that, and it didn't work. The deer still revived.

**Comment by NetSysFire** [comment]
Hmm, it looks like some handling for small dogs in `mammal.json` use delete:

https://github.com/CleverRaven/Cataclysm-DDA/blob/be2f42b2e33aea4ba70b245cc0284c5fcedbd7e5/data/json/monsters/mammal.json#L1140

**Comment by MNG-cataclysm** [comment]
> Hmm, it looks like some handling for small dogs in `mammal.json` use delete:
> 
> https://github.com/CleverRaven/Cataclysm-DDA/blob/be2f42b2e33aea4ba70b245cc0284c5fcedbd7e5/data/json/monsters/mammal.json#L1140

I also tried that, and it did not work.

**Comment by NetSysFire** [comment]
Weeeeird. Now i gotta test if beagles revive. For science.

**Comment by MNG-cataclysm** [comment]
> Weeeeird. Now i gotta test if beagles revive. For science.

Beagles do revive as zombie dogs, as do chihuahuas.

**Comment by Karol1223** [comment]
> > Weeeeird. Now i gotta test if beagles revive. For science.
> 
> Beagles do revive as zombie dogs, as do chihuahuas.

Wait a second they _do_? So you're saying that when it comes to copy_from the zombifies_into line isn't able to be removed in any way shape or form? Because I'm the author of the PR that specifically aimed to make Chihuahuas and Beagles NOT revive. That sounds like it shouldn't work like this...

**Comment by MNG-cataclysm** [comment]
> Wait a second they _do_? So you're saying that when it comes to copy_from the zombifies_into line isn't able to be removed in any way shape or form? Because I'm the author of the PR that specifically aimed to make Chihuahuas and Beagles NOT revive. That sounds like it shouldn't work like this...
> 
> 
> 
> EDIT: They absolutely do still zombify. And this is absolutely not intended behavior. That warrants opening a bug report. Will you take care of that or should I do it?

I think you should. You seem to know more about the whole problem than I do.

**Comment by terryjacob200** [comment]
I was testing this last night when trying to fix the dermatik host issue.  It seemed like "delete" would not remove "zombify_into" properly.  I did get "zombify_into" : "mon_null" to stop revive but the corpse items still had the yellow text as though they would revive.

**Comment by I-am-Erk** [comment]
I don't really doing it this way, I think this can wait until we fix the copy-from problem. Also why is this PR editing map extras?

**Comment by MNG-cataclysm** [comment]
> I don't really doing it this way, I think this can wait until we fix the copy-from problem. Also why is this PR editing map extras?

That was to resolve an error about furniture being placed on top of one another in some Exodii drop mapgen.

**Review by Maleclypse — DISMISSED** [review]
Please put the map extra flags in a separate PR that can go in immediately.

**Comment by anothersimulacrum** [comment]
I would suggest they revive in the flesh cocoons https://github.com/CleverRaven/Cataclysm-DDA/pull/60605

**Comment by MNG-cataclysm** [comment]
It seems that someone else already fixed it.

**Pull request closed without merging**